drizzlepac/haputils/svm_quality_analysis.py

```diff
@@ -1538,6 +1538,22 @@ def build_svm_plots(data_source, output_basename=''):
 # Functions for generating each data plot
 #    
 def build_gaia_plots(gaiaDF, data_cols, output_basename='svm_qa'):
+    """
+    Generate the plots for evaluating the distribution of GAIA catalog sources 
+    in the field-of-view of each product.
+    
+    Parameters
+    ----------
+    gaiaDF : Pandas ColumnDataSource 
+        This object contains all the columns relevant to the plots.
```

Comment:
The variable name is misleading if this is really a ColumnDataSource object. Please adjust the name or clarify the description.

Reply:
Renamed to 'gaiaCDS' in 7ba7497.

Reply:
Docstring here and in other functions were also updated to reflect the new names